### PR TITLE
BEI-29: Create the Sequelize migration and model files for the recipe…

### PR DESCRIPTION
**Pull request by LucianaDera**
WHY THIS PR?
We need a Sequelize migration and model files to define the structure for the recipe ingredient table.

WHAT DOES THIS PR DO?
This step allows Sequelize to recognize the model and map the recipe ingredient to the respective table in the database.

HOW CAN THIS PR BE TESTED?
N/A

THINGS YOU MAY WANT THE REVIEWER TO NOTE.
N/A

### CODE REVIEW

File: src/core/database/migrations/20230728203815-create_recipe_ingredients_table.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      createdAt: {
+        type: Sequelize.DATE,
+        allowNull: false,
+      },
+      updatedAt: {
+        type: Sequelize.DATE,
+        allowNull: false,
```
Comment (hizikex) [inline src/core/database/migrations/20230728203815-create_recipe_ingredients_table.js:30]: These should be in snake case

File: src/core/interfaces/recipe.ingredient.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export interface RecipeIngredientAttributes {
+  ingredient_id: number;
+  recipe_id: number;
+  quantity: number;
+  unit: string;
+}
```
Comment (hizikex) [inline src/core/interfaces/recipe.ingredient.ts:6]: NO extra line here

File: src/core/models/recipe.ingredient.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  {
+    sequelize,
+    tableName: 'recipeIngredients',
+  }
+);
+
+RecipeIngredient.belongsTo(Recipe, {
+  foreignKey: 'recipeId',
+  as: 'recipe',
+});
```
Comment (hizikex) [inline src/core/models/recipe.ingredient.ts:43]: No extra line

**Review by hizikex — CHANGES_REQUESTED** [review]

File: src/core/models/recipe.ingredient.ts (hunk 3 above)
Comment (LucianaDera) [reply src/core/models/recipe.ingredient.ts:43]: They are extra lines already

**Review by RICHYDOS — APPROVED** [review]

**Review by hizikex — APPROVED** [review]

**Pull request merged**